myft/main.scss

```diff
@@ -71,6 +71,53 @@ $_n-ui-myft-is-silent: nUiHas('myft') == false or $_n-ui-myft-applied == true;
 		}
 	}
 
+	.n-myft-ui--prefer-group {
+		@include oGridRespondTo($from: 'M') {
+			padding-left: $spacing-unit;
+		}
```

Comment:
Ah, it's just a warning but I'll amend the ordering. It wants this:
`@extends`, `@includes` without `@content`, properties, `@includes` with `@content`